packages/cdktf-cli/bin/cmds/ui/components/bottom-bars/approve.tsx

```diff
@@ -1,9 +1,64 @@
 import React from "react";
-import { Confirm } from "../confirm";
+import SelectInput from "ink-select-input";
+import { Box, Text } from "ink";
 
 type Props = {
   onApprove: () => void;
+  onDismiss: () => void;
+  onStop: () => void;
 };
-export function ApproveBottomBar({ onApprove }: Props) {
-  return <Confirm onApprove={onApprove} />;
+
+type Option = { label: string; description: string };
+type TransformedOption = Option & { value: string };
+const options: Record<string, Option> = {
+  approve: {
+    label: "Approve",
+    description: `Applies the changes outlined in the plan.`,
+  },
+  dismiss: {
+    label: "Dismiss",
+    description: `Does not apply the changes outlined in the plan. This will also prevent depending stacks from planning.`,
+  },
+  stop: {
+    label: "Stop",
+    description: `Does not apply the changes. Currently running stacks will be finished, but no new ones will be started.`,
```

Comment:
What is the exact difference between `stop` and `dismiss`? Is it that `dismiss` only stops the current branch of execution where `stop` stops the entire application deployment?

Reply:
Yeah exactly. Stop is a nicer /  needed because a ctrl-c would kill running applies (if we have parallelism) so you'd have to individually dismiss every single plan.

Reply:
No, stopping is pretty immediate. Plus you see all the output so you have a way to tell. Showing the status of all the stacks in progress is part of the parallelism work, so it's next up :)